app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= flash[:message] %>
+
+<h1>Sing In</h1>
+
+ <%= form_for :session, url:signin_path do |f|%>
+  <%= f.label :private_event_user_id, 'User ID'%>
+  <%= f.text_field :private_event_user_id, class:'form-control' %>
+
+  <%= f.submit "Submit" %>
+<% end %>
```

Comment:
How do  I know the ID I'm supposed to log in with? I tried with email and the name I registered with, didn't work.

<img width="361" alt="Screenshot 2020-06-05 at 11 11 52" src="https://user-images.githubusercontent.com/31304943/83865018-7bbb8b00-a71d-11ea-9a8c-3a99cd708aa5.png">